<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1389</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2E2E2E;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="ChartDisplayOptionsForm">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background-color: #1e1e1e;
    border: 1px solid #00CFFF;
    border-radius: 8px;
}

QLabel {
    color: #00CFFF;
    background-color: transparent;
    font-family: 'Segoe UI';
    font-size: 14px;
    border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_form3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #00CFFF;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
}

QPushButton:hover {
    background-color: #1ABC9C;
    color: black;
}

QPushButton:pressed {
    background-color: #16A085;
    border: 1px solid #16A085;
}
</string>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusIcon_form3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: white ;
    border-radius: 10px;
    border: 1px solid black;
    min-width: 20px;
    min-height: 20px;
    max-width: 20px;
    max-height: 20px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_FPSValue">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00AAFF;
font-size: 18px;
height: 6px;

margin: 0px;
border-radius: 3px;

  </string>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 18px;;
height: 6px;
margin: 0px;
border-radius: 3px;

color: #00CFFF;
background-color: transparent;
</string>
     </property>
     <property name="text">
      <string>FPS :</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_FPS">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>137</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #444;
    height: 6px;
    background: #2a2a2a;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    background: #00CFFF;
    border: none;
    width: 14px;
    height: 14px;
    margin: -4px 0;
    border-radius: 7px;
}

QSlider::sub-page:horizontal {
    background: #00CFFF;
    border-radius: 3px;
}

QSlider::add-page:horizontal {
    background: #444;
    border-radius: 3px;
}
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_LineICON">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_BarIcon">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radiolineChart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioBarChart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="DataFilterForm">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background-color: #1e1e1e;
    border: 1px solid #00CFFF;
    border-radius: 8px;
}

QLabel {
    color: #00CFFF;
    background-color: transparent;
    font-family: 'Segoe UI';
    font-size: 14px;
    border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBox_Max">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>16</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    color: #00CFFF;
    spacing: 8px;
    font-family: 'Segoe UI';
    font-size: 14px;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border-radius: 4px;
    border: 2px solid #00CFFF;
    background-color: #2a2a2a;
}

QCheckBox::indicator:checked {
    image: none;
    background-color: #00CFFF;
    border: 2px solid #00AACC;
}

QCheckBox::indicator:unchecked:hover {
    border: 2px solid #F39C12;
    background-color: #2a2a2a;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Min_form4">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>47</y>
       <width>130</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #2a2a2a;
    color: #00CFFF;
    border: 1px solid #00CFFF;
    border-radius: 6px;
    padding: 6px 8px;
    font-family: 'Segoe UI';
    font-size: 14px;
    selection-background-color: #00CFFF;
    selection-color: #1e1e1e;
}

QLineEdit:hover {
    border: 1px solid #F39C12;
}

QLineEdit:focus {
    border: 1px solid #1ABC9C;
    background-color: #1e1e1e;
}
</string>
     </property>
     <property name="placeholderText">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Max_form4">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>12</y>
       <width>130</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #2a2a2a;
    color: #00CFFF;
    border: 1px solid #00CFFF;
    border-radius: 6px;
    padding: 6px 8px;
    font-family: 'Segoe UI';
    font-size: 14px;
    selection-background-color: #00CFFF;
    selection-color: #1e1e1e;
}

QLineEdit:hover {
    border: 1px solid #F39C12;
}

QLineEdit:focus {
    border: 1px solid #1ABC9C;
    background-color: #1e1e1e;
}
</string>
     </property>
     <property name="placeholderText">
      <string>Max</string>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="colorCombox_2_Min">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>47</y>
       <width>85</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    background-color: #2a2a2a;
    color: #00CFFF;
    border: 1px solid #00CFFF;
    border-radius: 6px;
    padding: 4px 8px;
}
QComboBox:hover {
    border: 1px solid #F39C12;
}
QComboBox QAbstractItemView {
    background-color: #1e1e1e;
    border: 3px solid #00CFFF;
    selection-background-color: #00CFFF;
    selection-color: #1e1e1e;
    color: #00CFFF;
    outline: 0;
}</string>
     </property>
    </widget>
    <widget class="QComboBox" name="colorCombox_1_Max">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>12</y>
       <width>85</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    background-color: #2a2a2a;
    color: #00CFFF;
    border: 1px solid #00CFFF;
    border-radius: 6px;
    padding: 4px 8px;
    font-family: 'Segoe UI';
    font-size: 14px;
}
QComboBox:hover {
    border: 1px solid #F39C12;
}
QComboBox QAbstractItemView {
    background-color: #1e1e1e;
    border: 1px solid #00CFFF;
    selection-background-color: #00CFFF;
    selection-color: #1e1e1e;
    color: #00CFFF;
    outline: 0;
}</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Min">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>51</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    color: #00CFFF;
    spacing: 8px;
    font-family: 'Segoe UI';
    font-size: 14px;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border-radius: 4px;
    border: 2px solid #00CFFF;
    background-color: #2a2a2a;
}

QCheckBox::indicator:checked {
    image: none;
    background-color: #00CFFF;
    border: 2px solid #00AACC;
}

QCheckBox::indicator:unchecked:hover {
    border: 2px solid #F39C12;
    background-color: #2a2a2a;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_form4_Filter">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>82</y>
       <width>130</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #00CFFF;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
}

QPushButton:hover {
    background-color: #1ABC9C;
    color: black;
}

QPushButton:pressed {
    background-color: #16A085;
    border: 1px solid #16A085;
}
</string>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_form4_Clean">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>82</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #00CFFF;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
}

QPushButton:hover {
    background-color: #1ABC9C;
    color: black;
}

QPushButton:pressed {
    background-color: #16A085;
    border: 1px solid #16A085;
}
</string>
     </property>
     <property name="text">
      <string>clean</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusIcon_form4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: white ;
    border-radius: 10px;
    border: 1px solid black;
    min-width: 20px;
    min-height: 20px;
    max-width: 20px;
    max-height: 20px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="DataSamplingForm">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>229</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background-color: #1e1e1e;
    border: 1px solid #00CFFF;
    border-radius: 8px;
}

QLabel {
    color: #00CFFF;
    background-color: transparent;
    font-family: 'Segoe UI';
    font-size: 14px;
    border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QRadioButton" name="radioNormal">
     <property name="geometry">
      <rect>
       <x>143</x>
       <y>23</y>
       <width>74</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Normal</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioMin">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>23</y>
       <width>53</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Range_form2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>124</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #2a2a2a;
    color: #00CFFF;
    border: 1px solid #00CFFF;
    border-radius: 6px;
    padding: 6px 8px;
    font-family: 'Segoe UI';
    font-size: 14px;
    selection-background-color: #00CFFF;
    selection-color: #1e1e1e;
}

QLineEdit:hover {
    border: 1px solid #F39C12;
}

QLineEdit:focus {
    border: 1px solid #1ABC9C;
    background-color: #1e1e1e;
}
</string>
     </property>
     <property name="placeholderText">
      <string>Range</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusIcon_form2_server">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: white ;
    border-radius: 10px;
    border: 1px solid black;
    min-width: 20px;
    min-height: 20px;
    max-width: 20px;
    max-height: 20px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioMax">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>23</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_form2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
/* ~~~~~~~~~~~~~~~
    QPushButton
 ~~~~~~~~~~~~~~~ */
 

 QPushButton {
 color: #00CFFF;
 font-size: 16px;
 font-weight: bold;
 background-color: #2C3E50;
 padding: 4px 8px;
 border-radius: 5px;
 }

 QPushButton:hover {
     background-color: #1ABC9C;
     color: black;
 }

 QPushButton:pressed {
     background-color: #16A085;
     border: 1px solid #16A085;
 }
</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="UserStatusForm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background-color: #1e1e1e;
    border: 1px solid #00CFFF;
    border-radius: 8px;
}

QLabel {
    color: #00CFFF;
    background-color: transparent;
    font-family: 'Segoe UI';
    font-size: 14px;
    border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="statusIcon_form1_server">
     <property name="geometry">
      <rect>
       <x>227</x>
       <y>90</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: white ;
    border-radius: 10px;
    border: 1px solid black;
    min-width: 20px;
    min-height: 20px;
    max-width: 20px;
    max-height: 20px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_UserName_Blue">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>12</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 18px;;
height: 6px;
margin: 0px;
border-radius: 3px;

color: #00CFFF;
background-color: transparent;
</string>
     </property>
     <property name="text">
      <string>User Name: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Usernname">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>12</y>
       <width>129</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;;
height: 6px;
margin: 0px;
border-radius: 3px;
color: #80D8C3;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_IP">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>38</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 18px;;
height: 6px;
margin: 0px;
border-radius: 3px;

color: #16A085;
background-color: transparent;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>64</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 18px;;
height: 6px;
margin: 0px;
border-radius: 3px;

color: #00CFFF;
background-color: transparent;
</string>
     </property>
     <property name="text">
      <string>&gt;&gt; </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_MessageIS">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>64</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 17px;;
height: 6px;
margin: 0px;
border-radius: 3px;
color: rgb(255, 214, 107);
background-color: transparent;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="chartFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>1371</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_LoggingForm">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>341</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background-color: #1e1e1e;
    border: 1px solid #00CFFF;
    border-radius: 8px;
}

QLabel {
    color: #00CFFF;
    background-color: transparent;
    font-family: 'Segoe UI';
    font-size: 14px;
    border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0" colspan="2">
      <widget class="QLineEdit" name="Passwor_inForm1">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #2a2a2a;
    color: #00CFFF;
    border: 1px solid #00CFFF;
    border-radius: 6px;
    padding: 6px 8px;
    font-family: 'Segoe UI';
    font-size: 20px;
    selection-background-color: #00CFFF;
    selection-color: #1e1e1e;
}

QLineEdit:hover {
    border: 1px solid #F39C12;
}

QLineEdit:focus {
    border: 1px solid #1ABC9C;
    background-color: #1e1e1e;
}
</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLineEdit" name="UserName_inform1">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #2a2a2a;
    color: #00CFFF;
    border: 1px solid #00CFFF;
    border-radius: 6px;
    padding: 6px 8px;
    font-family: 'Segoe UI';
    font-size: 20px;
    selection-background-color: #00CFFF;
    selection-color: #1e1e1e;
}

QLineEdit:hover {
    border: 1px solid #F39C12;
}

QLineEdit:focus {
    border: 1px solid #1ABC9C;
    background-color: #1e1e1e;
}
</string>
       </property>
       <property name="placeholderText">
        <string>User Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="Port_inForm1">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #2a2a2a;
    color: #00CFFF;
    border: 1px solid #00CFFF;
    border-radius: 6px;
    padding: 6px 8px;
    font-family: 'Segoe UI';
    font-size: 20px;
    selection-background-color: #00CFFF;
    selection-color: #1e1e1e;
}

QLineEdit:hover {
    border: 1px solid #F39C12;
}

QLineEdit:focus {
    border: 1px solid #1ABC9C;
    background-color: #1e1e1e;
}
</string>
       </property>
       <property name="placeholderText">
        <string>12345</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_UserName_Blue_2">
       <property name="styleSheet">
        <string notr="true">
font-size: 28px;;
height: 6px;
margin: 0px;
border-radius: 3px;

color: #F1C40F;
background-color: transparent;
</string>
       </property>
       <property name="text">
        <string>      Logging form</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_form1_Loggin">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::WheelFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #00CFFF;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
}

QPushButton:hover {
    background-color: #1ABC9C;
    color: black;
}

QPushButton:pressed {
    background-color: #16A085;
    border: 1px solid #16A085;
}
</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: #F1C40F;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 2px 2px;
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: #F1C40F;
font-size: 20px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="IP_inForm1">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #2a2a2a;
    color: #00CFFF;
    border: 1px solid #00CFFF;
    border-radius: 6px;
    padding: 6px 8px;
    font-family: 'Segoe UI';
    font-size: 20px;
    selection-background-color: #00CFFF;
    selection-color: #1e1e1e;
}

QLineEdit:hover {
    border: 1px solid #F39C12;
}

QLineEdit:focus {
    border: 1px solid #1ABC9C;
    background-color: #1e1e1e;
}
</string>
       </property>
       <property name="placeholderText">
        <string>127.0.0.2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1389</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
